app/src/user/user.service.ts

```diff
@@ -420,6 +491,40 @@ export class UserService {
         }
         `;
 
+        const removeUserTechnologies = `mutation {
+            delete_user_technology(
+                where: {
+                    user_id: {_eq: "${userId}"}
+                },
+            ) {
+                returning {
+                    user_id
+                    technology_id
+                }
+            }
+        }`;
+
+        let userTechnologiesTemp = [];
+        if (technologies.length) {
+            userTechnologiesTemp = technologies.map(el => {
+                return `{
+                    user_id: "${userId}",
+                    technology_id: "${el}",
+                }`;
+            });
+        }
+
+        const insertUserTechnologies = `mutation {
+            insert_user_technology(
+                objects: [${userTechnologiesTemp}]
```

Comment:
Replace with `objects: ${technologies.map(el => ({ user_id: '"' + userId + '"', technology_id: '"' + el + '"' }))}`. Then you could remove lines from 507 to 515

Reply:
it doesn’t work like that, you need a string, and this returns an object,
works like that: objects: [${technologies.map(el => { user_id: "${userId}", technology_id: "${el}" })}]